sbin/dumpon/dumpon.c

```diff
@@ -487,6 +493,7 @@ main(int argc, char *argv[])
 				errx(EX_USAGE, "Unrecognized cipher algorithm "
 				    "'%s'", optarg);
 			break;
+		#endif
```

Comment:
We should print a more useful error message here if `-C` isn't implemented. I suggest adding `errx(EX_USAGE, "built without crypto support, -C is unhandled");` or so.